hw2/04-background.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>04 - Background</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+    <script defer src="04-background.js"></script>
+  </head>
+  <body id="body-color">
+    <main>
+      <div class="bootstrap-grid" id="bootstrap-grid">
+        <div class="row">
+          <div class="container col-12 col-sm-7 col-md-6 col-lg-5 rounded bg-white" id="color-change-card">
+              <h1 style="margin-bottom: 20px; margin-top: 10px;">Exercise 04 - Background</h1>
+              <p>
+                The background color will change every few seconds (as indicated below).
+                You can stop this behavior and you can also change how often the
+                background color changes.
+              </p>
+              <div class="number-entry-prompt">
+                <label for="num-input">Background change interval (seconds):</label>
+              </div>
+              <div class="user-num-input">
+                <input id="num-input" type="number" label="user input" />
+              </div>
+              <button id="toggle-button" type="button" class="btn" style="margin-top: 20px; padding-left: 15px; padding-right: 15px; padding-top: 5px; padding-bottom: 5px;">Stop</button>
```

Comment:
You should use bootstrap classes.

Reply:
Can you be more specific? This seems to be a style opinion as I've used bootstrap classes elsewhere.